src/main/java/seedu/address/logic/commands/SortEventCommand.java

```diff
@@ -0,0 +1,128 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Comparator;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.event.Event;
+
+/**
+ * Sorts the currently displayed persons in a specific order.
+ * Index entered determines the specific order.
+ */
+public class SortEventCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort-e";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sorts the currently displayed list of events "
+            + "by the index command entered\n"
+            + "1 will be sort by alphabetical order of the events' descriptions\n"
+            + "2 will be sort by alphabetical order of the events' time\n"
+            + "3 will be sort by alphabetical order of the number of participants of the events\n"
+            + "Parameters: INDEX (must be between 1 and 2) "
+            + "Example: " + COMMAND_WORD + " 1 ";
+
+    public static final String MESSAGE_ARGUMENTS = "Index: %1$d";
+
+    private static final Comparator<Event> DESCRIPTION_COMPARATOR = new Comparator<Event>() {
+        @Override
+        public int compare(Event o1, Event o2) {
+            return o1.getDescription().fullDescription.compareToIgnoreCase(o2.getDescription().fullDescription);
+        }
+    };
+
+    private static final Comparator<Event> TIME_COMPARATOR = new Comparator<Event>() {
+        @Override
+        public int compare(Event o1, Event o2) {
+            return o1.getTime().time.compareTo(o2.getTime().time);
+        }
+    };
+
+    //    private static final Comparator<Event> TAG_COMPARATOR = new Comparator<Event>() {
+    //        @Override
+    //        public int compare(Event o1, Event o2) {
+    //            if (o1.getTags().size() == 0 && o2.getTags().size() != 0) {
+    //                return 1;
+    //            }
+    //            if (o1.getTags().size() != 0 && o2.getTags().size() == 0) {
+    //                return -1;
+    //            }
+    //            if (o1.getTags().size() == 0 && o2.getTags().size() == 0) {
+    //                return o1.getName().fullName.compareToIgnoreCase(o2.getName().fullName);
+    //            }
+    //            return o1.getTags().iterator().next().tagName
+    //                    .compareToIgnoreCase(o2.getTags().iterator().next().tagName);
+    //        }
+    //    };
```

Comment:
Related to the above comment, to add this to a separate branch and subsequent PR so it is obvious to us what the changes are in the future PR.

Reply:
Have removed the third implementation for now as I've already implemented the feature with the tag in another branch.